vars/parseStageInfo.groovy

```diff
@@ -148,6 +148,10 @@ def call(Map config = [:]) {
       result['ftest_arg'] = config['ftest_arg']
     }
 
+    if (env.STAGE_NAME.contains('NLT')) {
+      result['NLT'] = 1
```

Comment:
This seems like a boolean, but moreover, shouldn't rely on unset values.
```suggestion
      result['NLT'] = True
    } else {
        result['NLT'] = False
```

Reply:
Agreed.  I was basing it on the 'with_valgrind' setting which relies on unset values.  Not being familiar with Groovy I wasn't confident to try booling vars, will make changes as suggested.

Reply:
> Not being familiar with Groovy I wasn't confident to try booling vars

I'm only just getting familiar with it so some of my suggestions might not actually work.  But just thinking we should try to embrace the language as much as we can and use it's types when we can.

Reply:
Agreed.  I was just trying the simplest thing to get it working initially.